src/Administration/Resources/app/administration/src/module/sw-sales-channel/view/sw-sales-channel-detail-base/index.js

```diff
@@ -744,7 +744,7 @@ export default {
                 paymentSettingsLink: routeData.href,
             };
 
-            return this.$tc(snippet, collection.length, data);
+            return this.$tc(snippet, data, collection.length);
```

Comment:
You can (and should) use $t instead. $tc is deprecated in the original vue dependency. Currently, $tc is only reference to $t anyway.